src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionPatcher.java

```diff
@@ -59,13 +63,15 @@ public DissolutionPatchResponse addDirectorApproval(String companyNumber, String
         return this.responseMapper.mapToDissolutionPatchResponse(companyNumber);
     }
 
-    public void updatePaymentInformation(String paymentReference, Timestamp paidAt, String companyNumber) {
+    public void handlePayment(String paymentReference, Timestamp paidAt, String companyNumber) {
         final Dissolution dissolution = this.repository.findByCompanyNumber(companyNumber).get();
 
         this.addPaymentInformation(paymentReference, paidAt, dissolution);
 
         setDissolutionStatus(dissolution, ApplicationStatus.PAID);
 
+        dissolution.setSubmission(this.dissolutionSubmissionMapper.generateSubmissionInformation());
```

Comment:
Can we move `this.dissolutionSubmissionMapper.generateSubmissionInformation()` to a separate variable above? Makes this more readable in my opinion.

Reply:
Disagree, but happy to hear other opinions